rust/crd/src/airflowdb.rs

```diff
@@ -61,13 +62,14 @@ impl AirflowDB {
                 .with_recommended_labels(build_recommended_labels(
                     airflow,
                     AIRFLOW_DB_CONTROLLER_NAME,
-                    version,
+                    &resolved_product_image.product_version,
                     "db-initializer",
                     "global",
                 ))
+                .with_label(APP_VERSION_LABEL, &resolved_product_image.app_version_label)
```

Comment:
Any particular reason to not pass `resolved_product_image.app_version_label` into `build_recommended_labels`? At the first glance it seems as this statements overwrite the recommended label.

Reply:
Thanks good point https://github.com/stackabletech/airflow-operator/pull/206/commits/2c4c9915e8e923338d549d3d10b4d1dc968654fa